lib/hammer_cli/logger.rb

```diff
@@ -54,7 +54,7 @@ def self.initialize_logger(logger)
       log_dir = File.expand_path(HammerCLI::Settings.get(:log_dir) || DEFAULT_LOG_DIR)
       begin
         FileUtils.mkdir_p(log_dir, :mode => 0750)
-      rescue Errno::EACCES => e
+      rescue Errno::EACCES => _
```

Comment:
Repeated pattern, so only commenting once.
```suggestion
      rescue Errno::EACCES
```